src/components/rangeslider/attributes.js

```diff
@@ -0,0 +1,47 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
+    visible: {
+        valType: 'boolean',
+        dflt: false,
+        description: 'Determines whether or not the range slider will be visible.'
+    },
+    bordercolor: {
+        valType: 'color',
+        dflt: 'transparent',
+        role: 'style',
+        description: 'Sets the border color of the range slider.'
+    },
+    borderwidth: {
+        valType: 'integer',
+        dflt: 1,
```

Comment:
I'd vote for a default of `0`. That seems more consistent with our axes defaults.

Better yet, we could have a smart default, where `borderwidth` inherits from `xaxis.linewidth`.